convert2rhel/pkghandler.py

```diff
@@ -657,7 +657,7 @@ def install_gpg_keys():
     gpg_keys = [os.path.join(gpg_path, key) for key in os.listdir(gpg_path)]
     for gpg_key in gpg_keys:
         try:
-            restorable_key = RestorableRpmKey(gpg_key)
+            restorable_key = backup.RestorableRpmKey(gpg_key)
```

Comment:
Given we don't import `backup` by itself, does this work?
At the top of the file I see this
```py
from convert2rhel.backup import RestorableFile, remove_pkgs
```

Reply:
Line 27 imports `backup`.

(I didn't see that when I originally coded this either.)